.github/workflows/5_builderpackage_indexer.yml

```
@@ -338,12 +338,12 @@ jobs:
         with:
           aws-region: "us-east-1"
           role-to-assume: ${{ secrets.AWS_IAM_OVA_ROLE }}
-          role-session-name: "OVA-Builder"
+          role-session-name: "5_package_builder_indexer"
```

Replace hardcoded value with variable for better maintenance.
```suggestion
          role-session-name: ${{ github.workflow }} 
```